lib/tasks/sar.rake

```diff
@@ -3,7 +3,7 @@ namespace :sar do
   task :create => :environment do
     category = get_sar_category
     Timecop.freeze Time.now - 24.hours do
-      kase = Case::SAR::NonOffender.new
+      kase = Case::SAR.new
       kase.name = Faker::Name.name
       kase.email = Faker::Internet.email(kase.name)
       kase.subject = Faker::Company.catch_phrase
```

Comment:
We don't need sar.rake anymore now we can create sar cases through the interface.

Reply:
Hrm. We don't have the admin pages working yet, maybe this is worth keeping until that's done?